src/lsp/cobol_lsp/lsp_request.ml

```diff
@@ -133,6 +133,55 @@ let handle_get_project_config_command param registry =
     Lsp_error.invalid_params "param = %s (association list with \"uri\" key \
                               expected)" Yojson.Safe.(to_string (param :> t))
 
+let handle_get_cfg registry params =
+  let params = Jsonrpc.Structured.yojson_of_t params in
+  let uri, name, options = Yojson.Safe.Util.(
+      to_string @@ member "uri" params,
+      to_string @@ member "name" params,
+      try to_assoc @@ member "render_options" params with Type_error _ -> [])
+  in
+  let textDoc = TextDocumentIdentifier.create ~uri:(DocumentUri.of_path uri) in
+  try_with_main_document_data registry textDoc
+    ~f:begin fun ~doc:_ checked_doc ->
+      let jsoono =
+        Lsp_cfg.doc_to_cfg_jsoono ~filename:uri ~name ~options checked_doc
+      in Some jsoono
+    end |>
+  Option.get
```

Comment:
Not sure why there's a need for `Option.get` here. Like most other requests the payload of replies should also be made optional to account for improper uses of the requests, and those cases dealt with on the client side.

(That'll be fixed later though)